alloy-core/src/main/java/org/starchartlabs/alloy/core/Preconditions.java

```diff
@@ -181,6 +264,33 @@ public static void checkState(boolean expression, @Nullable Object errorMessage)
         }
     }
 
+    /**
+     * Evaluates a value against a provided {@link Predicate} expression
+     *
+     * @param value
+     *            An assignable value to evaluate
+     * @param predicate
+     *            An operation to evaluate against {@code value}
```

Comment:
The `@param<T>` should be moved to be after the other method arguments

Reply:
moved param<T>